### PR TITLE
Proposed COC for Solo community

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* Post non relevant spam contents
+* Attempting to poach solo employees
```
Comment (linsun) [inline CODE-OF-CONDUCT.md]: This doc is pretty much reuse Contributor Covenant 2.0, these 2 bullets are added

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+* Practicing [Slack etiquette](https://slack.com/blog/collaboration/etiquette-tips-in-slack) when interacting with community members via Slack
```
Comment (linsun) [inline CODE-OF-CONDUCT.md:31]: This is added on top of Contributor Covenant 2.0

File: CODE-OF-CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+Examples of unacceptable behavior include:
+
+* Post non relevant spam contents
```
Comment (nfuden) [inline CODE-OF-CONDUCT.md]: nit: Unsure what Relevant spam contents would be.
Therefore I propose something more like the following (though most likely not exactly the following)
```suggestion
* Posting content that is non-relevant, unhelpful, or solicitous (spam)
```

Comment (sam-heilbron) [inline CODE-OF-CONDUCT.md]: I like this reference to spam comments. Do you think it's worth it to include a descrition specifying what we mean by spam?

File: CODE-OF-CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
+# Solo Community Code of Conduct
+
+All members of the Solo community must abide by the following Code of Conduct,
+adapted from the [Contributor Covenant][homepage]. Only by respecting each
```
Comment (jenshu) [inline CODE-OF-CONDUCT.md]: is this missing a link?

File: CODE-OF-CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
```
Comment (jenshu) [inline CODE-OF-CONDUCT.md:68]:
```suggestion
Instances of abusive, harassment, or otherwise unacceptable behavior may be
```

**Comment by nfuden** [comment]
I like this as a general set of guidelines for the community but I still wonder if we have some codification internally as to playbook for enforcement (per gdoc) and potentially a outwards facing set of potential consequences per go bridge code of conduct. 

No strong opinions but definitely a question I have.

File: CODE-OF-CONDUCT.md (hunk 4 above)
Comment (linsun) [reply CODE-OF-CONDUCT.md]: yep, resolved!

File: CODE-OF-CONDUCT.md (hunk 3 above)
Comment (linsun) [reply CODE-OF-CONDUCT.md]: @nfuden added a few things here, feel free to suggest anything else you think appropriate.

File: CODE-OF-CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior,  harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
+
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
```
Comment (jenshu) [inline CODE-OF-CONDUCT.md]: this link needs to be fixed too

File: CODE-OF-CONDUCT.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
```
Comment (jenshu) [inline CODE-OF-CONDUCT.md:130]: and this is the same link, right?

File: CODE-OF-CONDUCT.md (hunk 6 above)
Comment (linsun) [reply CODE-OF-CONDUCT.md]: fixed!

File: CODE-OF-CONDUCT.md (hunk 7 above)
Comment (linsun) [reply CODE-OF-CONDUCT.md:130]: yep!

**Comment by linsun** [comment]
@[nfuden](https://github.com/nfuden) let us track internally separately. This doc is for broader community and public facing.

**Pull request merged**